### PR TITLE
Adding license expression

### PR DESCRIPTION
**Pull request by pratik0316**
In this PR ,I added the License Expression Library to the DeltaCode.
Now when we activate the the virtual environment from the DeltaCode , we can have an access to the attributes and methods in the license expression library.
Screenshot from 2020-03-18 11-30-25
when we are doing source bin activate in the directory we can have access to the license code objects
Screenshot from 2020-03-18 11-36-42
This PR is addressing the issue #90  and #91

### CODE REVIEW

**Comment by pratik0316** [comment]
@majurg I added the license-expression library [https://github.com/nexB/license-expression](url) to DeltaCode , please review it.

File: etc/conf/dev/requirements_base.txt [hunk 1]
```diff
@@ -11,3 +11,4 @@ coverage
 pytest-cov
 codecov
 pytest-rerunfailures
+pluggy
```
Comment (steven-esser) [inline etc/conf/dev/requirements_base.txt:14]: Why is this added here?

File: src/deltacode/cli.py [hunk 2]
```diff
@@ -34,7 +34,6 @@
 from deltacode import __version__
 from deltacode.utils import deltas, get_notice, collect_errors
 
-
```
Comment (steven-esser) [inline src/deltacode/cli.py:37]: Why is this line removed?

**Review by steven-esser — CHANGES_REQUESTED** [review]
Why are there many added whl files and this test__pyahocorasick.py?

File: etc/conf/dev/requirements_base.txt (hunk 1 above)
Comment (pratik0316) [reply etc/conf/dev/requirements_base.txt:14]: @majurg  this was the extra requirements which was present in the License-expression Library,and regarding the .whl files it was present in the thirdparty folder of license expression library.I thaught these extra requirements and thirdparty .whl files would be required during configuration after adding license-expression library to the DeltaCode

Comment (steven-esser) [reply etc/conf/dev/requirements_base.txt:14]: You would only need to add `license_expression` to the `install_requires` in setup.py. 

The .whl files can be added to thirdparty, but these do not need to explicitly added to any requirements files

Comment (pratik0316) [reply etc/conf/dev/requirements_base.txt:14]: Yes @majurg ,it was my bad. Since .whl file of license_expression is already present in the thirdparty package it would be sufficient to add only `license_expression` in it `install_requires `in the `setup.py`
